integration/kubernetes/helm-chart/alluxio/values.yaml

```diff
@@ -290,7 +290,7 @@ fuse:
     # Extra environment variables for the fuse pod
     # Example:
     # JAVA_HOME: /opt/java
-  image: alluxio/alluxio-fuse
+  image: alluxio/alluxio
```

Comment:
@ssz1997 i will prefer to remove the `.Values.fuse.image` and `.Values.fuse.imageTag`
and change all the places that using the `.Values.fuse.image` and `.Values.fuse.imageTag` to  {{ .Values.image }}:{{ .Values.imageTag }} for easier to maintain in the future. Can you help create a new PR for these changes?

integration/kubernetes/helm-chart/alluxio/templates/fuse/daemonset.yaml 
        - name: alluxio-fuse
          image: {{ .Values.fuse.image }}:{{ .Values.fuse.imageTag }}

Reply:
Makes sense. I'll do it today.